docs/students/signing-up.md

```diff
@@ -1,5 +1,22 @@
 # Signing Up 
 
-How to sign up for Bible First Online? You need a signup key. Then you can sign up. No key, no enchilada.
+If a Bible First coach has invited you to sign up, then hopefully you've been given either a web address or a signup key. If you've been given a web address, just visit it and follow the instructions.
 
-Lorem nihil placeat adipisci voluptate qui Dolorem tempora aut id eveniet alias? Velit corrupti nam totam dolores nihil Omnis perspiciatis aut laboriosam nisi maiores Aspernatur aliquid inventore libero expedita magnam
+*Example web addresses:*
+```
+https://bfst.co/en/K7YN39
+https://biblefirst.online/en/K7YN39/sign_up
+```
+If you've been given a signup key, [click here](https://biblefirst.online/sign_up) to go to the generic signup form and enter the signup key when prompted.
+
+*An example signup key:*
+```
+K7YN39
+```
+If you'd like to sign up for Bible First, but don't have a web address or a signup key, [click here](mailto:hello@biblefirst.online?subject=Signup%20Key%20Request) to request a signup key.
```

Comment:
@joshukraine  Does this `mailto` link work for you?

Reply:
Yep, works great!